EffectsTest/Game.cpp

```diff
@@ -874,36 +882,43 @@ void Game::CreateDeviceDependentResources()
         m_skinnedEffectFogPPL->SetFogEnd(fogend);
 
         //--- EnvironmentMapEffect ---------------------------------------------------------
-        m_envmap = std::make_unique<EnvironmentMapEffect>(device, EffectFlags::None, pdAlpha);
+        m_envmap = std::make_unique<EnvironmentMapEffect>(device, EffectFlags::Fresnel, pdAlpha);
```

Comment:
The default parameter was ``bool fresnel = true``, so ``EffectFlags::Fresnel`` is needed to match that.